app/views/aliquots/_well_failing_aliquot.html.erb

```diff
@@ -1,6 +1,18 @@
-<%= presenter.form.check_box(id) if failable?(container) %>
+<%= presenter.form.check_box(id, disabled: !failable?(container), data: {
+  quadrant_index: container.quadrant_index,
+  prevent_quadrant_fail: prevent_quadrant_fail?(container)
+}) %>
```

Comment:
Disable the checkbox if we can't be failed, rather than not rendering it. This lets us re-enable it when we fail the whole quadrant.